.github/ISSUE_TEMPLATE/doc-breaking-change.md

```diff
@@ -28,6 +28,13 @@ Text in brackets are placeholders; replace the text with the requested informati
 
 <!-- Do not modify anything below this line -->
 
+## Affected APIs
```

Comment:
Can we change the instruction of Affected APIs section to using docids like the .NET Framework breaking change template as below? docid is unique string representative identifier for type, method, etc. It can be read programmatically.
### Affected APIs
[| List of APIs affected by change referred to using docids|]

// For details on how to generate docids, please look at other change files or
// see https://msdn.microsoft.com/library/fsbx0t7x.aspx for the spec.
//
// For example, if all members of System.Xml.XmlTextReader is affected, we would have the following item:
// * `T:System.Xml.XmlTextReader`
//
// Notice that only the type is included, as everything contained with it will be included as well. This works for any level of the type
// hierarchy and allows for whole namespaces to be included if needed.
//
// If no specific API is affected, use the following entry:
//  * Not detectable via API analysis

Reply:
I'd prefer not to, @Lxiamail.  DocIDs are problematic given that they have to precisely match the signature of a method in a precise format. They've been a predictable source of broken links when publishing breaking changes. From the viewpoint of authoring (and remember that this is an issue that will not be consumed by any existing tool), it's far easier to use a simple member signature without requiring a particularly precise format.

Reply:
@rpetrusha We talked offline. The plan is that the issue template will have a link to the effected API, then in the generated markdown breaking change doc, it will have a comment with the corresponding DocID. That will work for ApiPort tool to consume. I'm good with that plan.